GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
+            var vsVersionType = new string[] { "Professional", "Community" };
```

Comment:
> My goal was not to refactore everthing and not see my PR merged....

The goal is should be improving UX and leaving the codebase in a better state than you found it.